types/datadog-winston/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for datadog-winston 1.0
+// Project: https://https://github.com/itsfadnis/datadog-winston
```

Comment:
Typo in project url:
```suggestion
// Project: https://github.com/itsfadnis/datadog-winston
```

Reply:
Fixed